src/DGtal/geometry/tools/Hull2DHelpers.h

```diff
@@ -352,6 +358,162 @@ namespace DGtal
+     * @param[in] def definition of the thickness used in the estimation (i.e HorizontalVerticalThickness or EuclideanThickness)
+     * @param[out] antipodalEdgeP one point of the antipodal edge associated to the minimal value of convex hull thickness. 
+     * @param[out] antipodalEdgeQ one point of the antipodal edge associated to the minimal value of convex hull thickness. 
+     * @param[out] antipodalVertexR the vertex of the antipodal pair associated to the minimal value of convex hull thickness. 
+     *
+     * @warning The convex hull should be oriented in counter clockwise else it will return wrong result.
+     **/
+    template <typename ForwardIterator, 
+              typename TInputPoint >
+    double computeHullThickness(const ForwardIterator& itb, 
+                                const ForwardIterator& ite,
+                                const ThicknessDefinition& def,
+                                TInputPoint& antipodalEdgeP,
+                                TInputPoint& antipodalEdgeQ,
+                                TInputPoint& antipodalVertexR);
+
+
+    /**
+     * Computes the angle between the line (@a a,@a b) and (@a c,@a d)
+     * @param[in] a one of point defining the first line. 
+     * @param[in] b a second point defining the first line. 
+     * @param[in] c a third point defining the second line. 
+     * @param[in] d a third point defining the second line. 
+     
+     **/
+    template<typename TInputPoint>
+    inline
+    double getAngle(const TInputPoint& a, const TInputPoint& b,const TInputPoint& c,const  TInputPoint& d);
+    
+    /**
+     * Computes the thickness of an anti podal pair (represented by
+     * the segment [ @a p , @a q ] and vertex @a r) according the given
+     * distance @def definition.  
+     *
+     * If the distance definition is @HorizontalVerticalThickness, it
+     * returns the minimal distance between the vertical/horizontal
+     * projection of @r on ( @a p , @a q ).
+     *
+     * If the distance definition is @EuclideanThickness, it returns
+     * the distance between r and its projection on the line ( @a p ,@a q ).
+     *
+     * @param[in] p the first point of the edge anti podal pair.
+     * @param[in] q the second point of the edge anti podal pair.
+     * @param[in] r the vertex of the anti podal pair.
+     * @param[in] def definition of the thickness used in the estimation (i.e HorizontalVerticalThickness or EuclideanThickness).
+     *
+     **/
+    template<typename TInputPoint>
+    double getThicknessAntipodalPair(const TInputPoint& p, const TInputPoint& q, 
+                                     const TInputPoint& r, const ThicknessDefinition& def);
+
+    /**
+     * Computes the horizontal distance a point @c according the segment [ @a a , @a b ].
+     * (i.e the horizontal projection distance of @c on [ @a a , @a b ]). 
+     * @note if the segment [@a a, @a b] is horizontal (i.e @a a [1]==@a b[1]) then an infinite value is returned. 
```

Comment:
In fact I remove the comment in the .ih because the bool return value already indicates if the points is inside or not (and the uses can do what he wants)...